### DISCUSSION TITLE
Lu Wilson (todepond) -- TLDraw, Makereal, Dreamberd

### DISCUSSION THREAD

**Discussion by just-be-dev** [devtools-fm/devtools.fm#312] [Announcements]
## Background 

- https://x.com/TodePond
- https://www.todepond.com/

### Selected Works

- https://github.com/TodePond/DreamBerd
- https://github.com/TodePond/Arroost
- https://github.com/tldraw/make-real
- https://github.com/TodePond/CellPond

### Writing

I like too much to list it all

- https://www.todepond.com/wikiblogarden/my-name/todepond/
- https://www.todepond.com/wikiblogarden/work/how-i-get-money/
- https://www.todepond.com/wikiblogarden/meta/narrative/modernism/obsession/

## Questions

- [x] Can you tell us the story of how team fortress 2 lead to todepond?
- [x] Even outside your work at tldraw you are incredibly creative when it comes to making things when coding. Where do you get inspired for all these ideas? What's one of your favorite projects you've published?
- [x] You've got a post talks about how you hold counter views to Bret Victor that's described through a modernist/post-modernist lens. You have this one quote that I love _"Sorry reader, better tools won’t save your shitty drawings"._ Can you talk a little about philosophy when it comes to tools and technology?

### TLDraw

- [x] How would you explain your role at TLDraw?
- [x] You and the TLDraw team have been showcasing a lot of AI demos. Make real, visual auto complete, a visual math interpreter, etc. Is there a specific product play that the team is making or is this more research, exploration, and (incidentally) damn good marketing?
- [x] It's been awhile since we talked to steve. What unique applications of tldraw have you seen?

### Ink and Switch

- [x] You're a researcher in residence at ink and switch. What sort of research do have you done with them so far?
- [ ] Whats the coolest thing you've seen there? 

### Future of Coding

- [x] You're active in the future of coding space. What draws you to this space and what are your hopes for it?

### Future

- [x] What do you think the future of software interaction looks like? AI is changing the way humans think about interacting with a computer, how might UI change to accommodate that?

**Comment by just-be-dev** [comment]
Tooltips:
- https://mitpress.mit.edu/9780262547482/building-simcity/
- https://toolstash.com/

**Comment by TodePond** [comment]
Tooltips:
https://saladroom.net/
https://github.com (!) (for sharing gifs and videos without compression)
https://getkap.co/

**Comment by hipstersmoothie** [comment]
https://waku.gg

**Reply by hipstersmoothie** [reply]
turbo v2
